packages/protect/src/lib/protect.ts

```diff
@@ -41,202 +41,217 @@ declare global {
  * @param {ProtectConfig} options - the configuration options for the PingOne Signals SDK
  * @returns {Promise<Protect>} - a set of methods to interact with the PingOne Signals SDK
  */
-export async function protect(options: ProtectConfig): Promise<Protect> {
-  try {
-    /*
-     * Load the Ping Signals SDK
-     * this automatically pollutes the window
-     * there are no exports of this module
-     */
-    await import('./signals-sdk.js' as string);
-    return {
-      start: async (): Promise<void> => {
-        await window._pingOneSignals.init(options);
-
-        if (options.behavioralDataCollection === true) {
-          window._pingOneSignals.resumeBehavioralData();
-        }
-      },
-      getData: async (): Promise<string> => {
-        return await window._pingOneSignals.getData();
-      },
-      pauseBehavioralData: (): void => {
-        window._pingOneSignals.pauseBehavioralData();
-      },
-      resumeBehavioralData: (): void => {
+// TODO: Handle initialization errors
+// TODO: changeset?
```

Comment:
I don't think we need a changeset for this as it's an unreleased module, but I do think we need to align all of these customer facing errors. Let's not do it in this PR, but I'm going to create a Jira ticket for error handling across all of these packages.

Reply:
Whoops, didn't mean to leave those comments in. I've removed them.